backend/db/utils.py

```diff
@@ -156,14 +156,24 @@ def is_table_up_to_date(table_name: str, skip_if_updated_within_hours: int = Non
     return check_if_updated(last_updated_key, skip_if_updated_within_hours)
 
 
-def update_db_status_var(key: str, val: str, local=False):
+def delete_db_status_var(key: str, local=False):
     """Update the `manage` table with information for a given variable, e.g. when a table was last updated
     todo: change to a 1-liner UPDATE statement"""
     with get_db_connection(schema='', local=local) as con2:
         run_sql(con2, f"DELETE FROM public.manage WHERE key = '{key}';")
+
+
+def update_db_status_var(key: str, val: str, local=False):
+    """Update the `manage` table with information for a given variable, e.g. when a table was last updated"""
+    with get_db_connection(schema='', local=local) as con2:
+        run_sql(con2, f"DELETE FROM public.manage WHERE key = '{key}';")
         sql_str = f"INSERT INTO public.manage (key, value) VALUES (:key, :val);"
         run_sql(con2, sql_str, {'key': key, 'val': val})
 
+def check_db_status_var(key: str,  local=False):
+    """Check the value of a given variable the `manage`table """
+    with get_db_connection(schema='', local=local) as con2:
+        return sql_query_single_col(con2, f"SELECT * FROM public.manage WHERE key = '{key}';")
```

Comment:
Thanks for adding this! One change to make is to change select from `SELECT *` to `SELECT value` (as in `last_refresh_timestamp()`).

Reply:
Hope probably made this change already. I should double check.

Reply:
Confirming change made! Thank you!